### PR TITLE
Adds pink and a custom color option to the circuit coloring tool

### PR DESCRIPTION
**Pull request by CalamaBanana**
The color pink was missing for the circuit detailer, completely unrelated to the newly added phallic assemblies

grafik

Edit:
Added in a custom color selection, too

### CODE REVIEW

**Comment by Useroth** [comment]
You could add a custom option while you're at it.
Something along the lines of...
`input(user, "Enter a color, please.", "Color", "#ffffff") as color|null`

**Comment by CalamaBanana** [comment]
> You could add a custom option while you're at it.

Managed to do it! Baby steps.

File: code/modules/integrated_electronics/core/detailer.dm [hunk 1]
```diff
@@ -25,7 +25,9 @@
 		"green" = COLOR_ASSEMBLY_GREEN,
 		"light blue" = COLOR_ASSEMBLY_LBLUE,
 		"blue" = COLOR_ASSEMBLY_BLUE,
-		"purple" = COLOR_ASSEMBLY_PURPLE
+		"purple" = COLOR_ASSEMBLY_PURPLE,
+		"pink" = COLOR_ASSEMBLY_PINK,
+		"custom" = "#ffffff"
```
Comment (Useroth) [inline code/modules/integrated_electronics/core/detailer.dm]:
```suggestion
		"custom" = COLOR_ASSEMBLY_WHITE
```

Comment (CalamaBanana) [reply code/modules/integrated_electronics/core/detailer.dm]: Fair enough, looks prettier, that color is never used though

**Review by deathride58 — APPROVED** [review]

**Pull request merged**